src/static/manifest.json

```diff
@@ -41,8 +38,20 @@
     "clipboardWrite",
     "storage"
   ],
-  "content_security_policy": "script-src 'self' 'wasm-eval'; object-src 'self';",
+  "host_permissions": [
+    "*://*/*"
+  ],
+  "content_security_policy": {
+    "extension_pages": "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'"
+  },
   "web_accessible_resources": [
-    "js/injectedScript.js"
+    {
+      "resources": [
+        "js/injectedScript.js"
+      ],
+      "matches": [
+        "*://*/*"
+      ]
+    }
   ]
-}
+}
```

Comment:
I highly recommend having the EditorConfig extension for the automatic configuration of basic formatting, like the trainling newline. This extension is a must for all BTE projects.

Reply:
Fixed at [4cb8865](https://github.com/BTE-Trusted-Entity/sporran-extension/pull/2632/commits/4cb8865be70f9645c14ee2fdee6ea32d2f536a03)